test/ResourceManager.test.cpp

```diff
@@ -21,3 +21,16 @@ TEST(ResourceManager, GetResourceStream_RefuseAbsolutePath) {
 	EXPECT_THROW(resourceManager.GetResourceStream("/Archive.vol"), std::runtime_error);
 	EXPECT_THROW(resourceManager.GetResourceStream("/PathTo/Archive.vol"), std::runtime_error);
 }
+
+TEST(ResourceManager, GetFilenames)
+{
+	// Ensure Directory.vol is not returned
+	ResourceManager resourceManager("./data");
+	auto filenames = resourceManager.GetAllFilenamesOfType(".vol");
+
+	EXPECT_EQ(0, filenames.size());
+
+	filenames = resourceManager.GetAllFilenames("Directory.vol");
+
+	EXPECT_EQ(0, filenames.size());
```

Comment:
This test can probably remove the temporaries. Ex:
```cpp
EXPECT_EQ(0, resourceManager.GetAllFilenames("Directory.vol").size());
```

Reply:
I don't have a real preference either way. Will make the change since you brought it up though.

Thanks for the quick review after I straightened out the branch rebase. Had to use the command line for that instead of tortoiseGit.